docs/zkapps/how-to-write-a-zkapp-ui.mdx

```diff
@@ -154,3 +153,91 @@ To see more details on deploying to Cloudflare Pages, [see their documentation o
   Cross-Origin-Opener-Policy: same-origin
   Cross-Origin-Embedder-Policy: require-corp
 ```
+
+### Connecting your zkApp with a user's wallet
+
+:::info
+
+Currently, only Auro Wallet's Chrome extension supports zkApp transactions. We
+anticipate other wallets to add support over time, which they can do by using
+the <a
+href="https://www.npmjs.com/package/auro-web3-provider">`auro-web3-provider`</a>
+from NPM within their browser-extension wallet.
+
+:::
+
+1. Install <a
+   href="https://chrome.google.com/webstore/detail/auro-walletmina-protocol/cnmamaachppnkjgnildpdmkaakejnhae">Auro
+   Wallet for Chrome</a>. Users of your zkApp will need to have this wallet
+   installed in order to interact with your zkApp. Once installed, `window.mina`
+   will automatically become available in the user's browser environment. Your
+   zkApp will then use this object to interact with the wallet.
+2. Get accounts - To fetch a user's list of Mina accounts, you can use the
```

Comment:
> 2. To fetch a list of Mina accounts in the user's wallet, you can use the `getAccounts()` method. This can be useful to indicate if the user's wallet is successfully connected to your zkApp, such as in the screenshot below.

I think this section might read more easily / more scan-able, by combining the text together, combining the two code snippets into one, and then finishing with your screenshot directly below the code snippet. might be clearer